5.6/s2i/bin/save-artifacts

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+pushd ${HOME} >/dev/null
+tar cf - vendor
+popd >/dev/null
```

Comment:
Git is complaining there about "No newline at end of fine". Please add it to `save-artifacts` files for all versions. Thanks.